spark/dl/src/main/scala/com/intel/analytics/bigdl/tensor/DnnStorage.scala

```diff
@@ -47,7 +47,7 @@ private[tensor] class DnnStorage[T: ClassTag](size: Int) extends Storage[T] {
   // Hold the address of the native array
   @transient var ptr: Pointer = new Pointer(allocate(size))
 
-  override def length(): Int = size
+  override def length(): Int = size / bytes
```

Comment:
add more comments to explain this length is not same with blas tensor lenght

Reply:
It's a wrong change. I'll restore it.